R/mfl_scoringhistory.R

```diff
@@ -0,0 +1,61 @@
+## ff_scoringhistory (MFL) ##
+
+#' Get a dataframe of scoring history, utilizing the ff_scoring and load_player_stats functions.
+#'
+#' @param conn a conn object created by \code{ff_connect()}
+#' @param season the season of interest back to 1999
+#' @param ... other arguments
+#'
+#' @examples
+#' \donttest{
+# ssb_conn <- ff_connect(platform = "mfl", league_id = 54040, season = 2020)
+# ff_scoringhistory(ssb_conn)
+#' }
+#'
+#' @seealso \url{http://www03.myfantasyleague.com/2020/scoring_rules#rules}
+#'
+#' @describeIn ff_scoringhistory MFL: returns scoring history in a flat table, one row per player per week.
+#'
+#' @export
+ff_scoringhistory.mfl_conn <- function(conn, season, ...) {
+
+  #Pull in scoring rules for that league
+  league_rules <-
+    ff_scoring(conn) %>%
+    tidyr::separate(col = range,
```

Comment:
I think you can just quote range into a string here, and same for using in the across function